example-apps/typescript/package.json

```diff
@@ -2,24 +2,20 @@
   "name": "typescript",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
+  "main": "dist/index.js",
   "scripts": {
-    "test": "npm run build && jest --testTimeout 10000 --rootDir ./lib/test",
+    "test": "vitest",
     "build": "npm run clean && tsc",
-    "clean": "rimraf ./lib ./build",
-    "watch": "npm run clean && tsc --watch",
+    "clean": "rimraf ./dist ./build",
     "_zapier-build": "npm run build"
   },
   "dependencies": {
     "zapier-platform-core": "15.12.0"
   },
   "devDependencies": {
-    "jest": "^25.5.3",
```

Comment:
Looks Jest was always being added because of `writeGenericPackageJson` ([source](https://github.com/zapier/zapier-platform/blob/abeac307ee5d24a2ecf35504a510906230df84ed/packages/cli/src/generators/index.js#L50)). I'll split out a TS+vitest specific template handler.